### ISSUE TITLE
Show result of motion threshold

### ISSUE THREAD

**Issue by alexyao2015** [blakeblackshear/frigate#2619]
**Describe what you are trying to accomplish and why in non technical terms**
I would like some way to determine visibly what each of the toggles under `motion:` is doing to figure out how to best adjust them.

**Describe the solution you'd like**
I would like some sort of live preview in the debug menu to show what it looks like or some way to feed in a recording to see what the processed image looks like and if motion is detected. 

**Describe alternatives you've considered**
Looked at the source code, manually picking out the commands, then running them locally.

**Additional context**
Reading the source code and contributing would be much easier if the code was fully typed. It really would help new people looking at the source code to understand what was happening. 

For example, [this was confusing to me](https://github.com/blakeblackshear/frigate/blob/bb68a2405b32bb821c69dbe2d9bd256083c99bc1/frigate/motion.py#L30) when I first looked at it because I didn't know what the type of `frame` was. Not sure if this is just my experience but I believe it would greatly help with readability.

**Comment by NickM-27** [comment]
The debug menu in cameras does have a toggle for motion boxes which would show you what motion is being detected.

**Comment by alexyao2015** [comment]
Yeah but that only tells me if something was detected or not. It doesn't show me what the program is seeing.
